lib/grizzly/zwave/crc.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Grizzly.ZWave.CRC do
+  @moduledoc """
+  CRC for Z-Wave commands
+
+  Some commands will need to use CRC for checking checksums. This module
+  exposes CRC functions to be able to run those checks.
+  """
+
+  @compile :native
+  @compile {:hipe, [:o3]}
```

Comment:
```suggestion
  @compile :native
  @compile {:hipe, [:o3]}
```

HiPE isn't supported on Nerves, so these won't help and should probably be deleted to avoid causing any confusion in the future.